README.md

```diff
@@ -18,3 +18,20 @@ The project is open source under the GNU Affero General Public License, with a p
 - Go to `https://tools.wmflabs.org/wikidata-todo/quick_statements.php`.
 - Authenticate yourself with your Wikidata account.
 - Copy and paste the contents of `quickstatement_commands.txt` into the blank field, and run the commands
+
+## Field mappings
+- bezeichnung -> P...
```

Comment:
Please only propose changes that can be accepted without needing additional changes. If you wish to propose that someone else makes a change, then raise an issue